.github/ISSUE_TEMPLATE/BUG.md

```diff
@@ -37,6 +37,16 @@ assignees: ''
 2. ...
 3. ...
 
+<!--
+     Please tell us on which platform(s) the problem occurs (Android / iOS / Web / Desktop)
```

Comment:
"Desktop" is not a platform, it's a category. The platforms are macOS, Linux, and Windows.

I also think this needs to be much more specific about asking what platform the app is being built for/run on, otherwise people are likely to give us the host platform (which we already know from `flutter doctor -v`)